commit 4b04ee5b50ef67df8636af837271984c4f7ead8b
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Thu Jun 20 22:00:10 2013 +0300

    Install the Grantlee filter into the right location.
    
    This is similar to what was done to Skrooge in commit 14baa01.
    
    Grantlee plugins, such as kdev_filters, are Qt plugins, and thus it makes
    sense to install it into ${PLUGIN_INSTALL_DIR} instead of installing into
    the same directory Grantlee uses for its own filters.
    
    It's also worth pointing out that people are expected to have the
    QT_PLUGIN_PATH environment variable correctly set for the filter to be
    found by Grantlee.

diff --git a/template/CMakeLists.txt b/template/CMakeLists.txt
index 0dfa658..a35d2cd 100644
--- template/CMakeLists.txt
+++ template/CMakeLists.txt
@@ -1,11 +1,3 @@
-
-set (
-    TEMPLATE_FILTERS_INSTALL_ARGS
-    LIBRARY DESTINATION "${LIB_INSTALL_DIR}/grantlee/${Grantlee_VERSION_MAJOR}.${Grantlee_VERSION_MINOR}"
-    ARCHIVE DESTINATION "${LIB_INSTALL_DIR}/grantlee/${Grantlee_VERSION_MAJOR}.${Grantlee_VERSION_MINOR}"
-    RUNTIME DESTINATION "${BIN_INSTALL_DIR}/grantlee/${Grantlee_VERSION_MAJOR}.${Grantlee_VERSION_MINOR}"
-)
-
 include (${Grantlee_USE_FILE})
 
 add_subdirectory(filters)
diff --git a/template/filters/CMakeLists.txt b/template/filters/CMakeLists.txt
index 8f59000..8a9e0a6 100644
--- template/filters/CMakeLists.txt
+++ template/filters/CMakeLists.txt
@@ -3,4 +3,4 @@ set(kdevfilters_SRCS kdevfilters.cpp)
 grantlee_add_plugin(kdev_filters kdevfilters)
 target_link_libraries(kdev_filters ${KDE4_KDECORE_LIBS} kdevplatformlanguage)
 
-install(TARGETS kdev_filters DESTINATION ${TEMPLATE_FILTERS_INSTALL_ARGS})
+install(TARGETS kdev_filters DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/grantlee/${Grantlee_VERSION_MAJOR}.${Grantlee_VERSION_MINOR})
